pyconfig.h.in

```diff
@@ -1850,7 +1853,7 @@
 /* Define to `unsigned int' if <sys/types.h> does not define. */
 #undef size_t
 
-/* Define to `int' if <sys/socket.h> does not define. */
+/* Define to 'int' if <sys/socket.h> does not define. */
```

Comment:
A little odd that *this* ``int'` changed, but others didn't.

Reply:
Vim's syntax highlighting poked me too much around that line.

Reply:
np, I'll revert it

Reply:
Reverted with 3deb15bc682a66f7685f666f582b3466deda1e30